packages/core/src/components/Image.story.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import Image from './Image';
+
+storiesOf('Core/Image', module)
+  .addParameters({
+    inspectComponents: [Image],
+  })
+  .add('default', () => (
+    <Image
+      src="https://images.freeimages.com/images/large-previews/035/young-golden-retriever-1404848.jpg"
+      alt="Something descriptive"
+      height={450}
+    />
+  ))
+  .add('background', () => (
+    <Image
+      background
+      src="https://images.freeimages.com/images/large-previews/035/young-golden-retriever-1404848.jpg"
+      alt="Something descriptive"
+      height={450}
+    />
+  ))
+  .add('background contain', () => (
+    <Image
+      background
+      backgroundSize="contain"
```

Comment:
for background size, we do boolean props like `cover` or `contain`. see ResponsiveImage for an example: http://airbnb.io/lunar/?path=/story/core-responsiveimage--with-contain

Reply:
(which should also be applied to `object-fit` if the src is not used as a background image)

Reply:
I see, I was following the logic that the DLS component used but this suggestion is a lot better. Fixed!😁